### DISCUSSION TITLE
Why does it still listen to port 8080 when the webui is disabled

### DISCUSSION THREAD

**Discussion by zeropool** [trinodb/trino#18817] [Q&A] [answered]
Why does it still listen to port 8080 when the webui is disabled? 
Also, I tried many conditions and cannot get it to listen to 127.0.0.1 as the webui port.

**Comment by hashhar** [comment, marked as answer]
Port 8080 is also the HTTP port where the Trino REST API is served. You cannot disable it otherwise you can't submit queries to Trino.
